research/conditional/utils/argparse.py

```diff
@@ -65,6 +65,7 @@ def introduce_parser_arguments(
         help="comma-separated list of integers, that signify the numbers of model blocks that are first on the new device, e.g. 2,4 means that blocks 0,1 will be on GPU 0, blocks 2,3 will be on GPU 1, and the rest will be on GPU 2",
     )
     parser.add_argument("--detect_anomaly", action="store_true")
+    parser.add_argument("--flash_attention", action="store_true")
```

Comment:
Hm, it's another parameter that really should be defaulted to True. It is ok for now, since it is consistent with our code, but we really need to change this in the future